src/taskgraph/run-task/run-task

```diff
@@ -1033,6 +1033,33 @@ def _display_python_version():
         b"setup", b"Python version: %s\n" % platform.python_version().encode("utf-8")
     )
 
+def _display_subprocess_python_version():
+    try:
+        output = subprocess.check_output(
+            ["uv", "python", "list", "--only-installed"],
+        ).strip()    
+
+        if output:
+            for line in output.splitlines():
+                print_line(b"setup", b"Subprocess python version: " + line + b"\n")
+
+    except FileNotFoundError:
```

Comment:
Does `FileNotFoundError` get raised? I thought it would be `subprocess.CalledProcessError`. You might need to change `FileNotFoundError` to that. Beware if you switch to `subprocess.run` above, you can pass `check=True` to make it raise an exception when the command fails.